bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -4578,134 +4578,148 @@
         </sql>
     </changeSet>
 
-    <changeSet id="bahmni-emrapi-202208101339" author="BAHMNI" >
+
+    <changeSet id="bahmni-emrapi-202208101439" author="BAHMNI" >
         <preConditions onFail="MARK_RAN">
             <sqlCheck expectedResult="1">
                 select count(*) from concept c
```

Comment:
instead of count(*) .. can you please use c.concept_id?

Reply:
Done